app/controllers/apps/evolution_apis_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Apps::EvolutionApisController < ActionController::Base
+  skip_before_action :verify_authenticity_token
```

Comment:
Hmmm aqui como é um endpoint que vai receber apenas requisições JSON de api talvez trocar o controller base seja mais interessante e também aplica otimizações nos cenários de API

```suggestion
class Apps::EvolutionApisController < Api::V1::PublicController
```

Reply:
Show!